### ISSUE TITLE
twisted.conch.client.knownhosts.PlainEntry misdocuments its "_hostnames" attribute.

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3548]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#3548|
|Type|defect|
|Created|2008-11-17 21:03:41Z|

The class docstring includes documentation for a non-existent `_hostname` attribute.

<details><summary>Searchable metadata</summary>

```
trac-id__3548 3548
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__conch conch
keywords__ 
time__1226955821000000 1226955821000000
changetime__1227014580000000 1227014580000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_1"></a>|@glyph commented|
|-|-|

```
#!diff
Index: twisted/conch/client/knownhosts.py
===================================================================
--- twisted/conch/client/knownhosts.py	(revision 25365)
+++ twisted/conch/client/knownhosts.py	(working copy)
@@ -95,8 +95,8 @@
     A L{PlainEntry} is a representation of a plain-text entry in a known_hosts
     file.
 
-    @ivar _hostname: the host-name associated with this entry.
-    @type _hostname: L{str}
+    @ivar _hostnames: the list of all host-names associated with this entry.
+    @type _hostnames: L{list} of L{str}
     """
 
     implements(IKnownHostEntry)
```

Look good?

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)|@glyph removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set owner to @glyph|
|-|-|

Yes, please apply.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_4"></a>|@glyph set status to `closed`|
|-|-|

(In [25411]) Correct the documentation of `PlainEntry` to document _hostnames, which exists, rather than _hostname, which does not.

Author: glyph
Reviewer: exarkun

Fixes [#3548](https://github.com/twisted/twisted/issues/3548)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
